index.html

```diff
@@ -9267,7 +9327,8 @@ <h2 id="defining-a-valid-audioworkletprocessor">
                   <code>Float32Array</code> of audio samples for the
                   <code>m</code>th channel of <code>n</code>th input. While the
                   number of inputs is fixed at the construction, the number of
```

Comment:
Not part of the PR, but noticed that "is fixed at the construction" -> "is fixed at construction"

Reply:
Done.